### PR TITLE
Add docker compose yml file

### PR DESCRIPTION
**Pull request by tobiasake**

### CODE REVIEW

File: src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java [hunk 1]
```diff
@@ -15,20 +16,25 @@
...
 
-public final class HttpExecutor {
+
+public class HttpExecutor {
 
     private static HttpExecutor instance;
     private CloseableHttpClient client = HttpClientBuilder.create().build();
     private static final Logger LOGGER = LoggerFactory.getLogger(HttpExecutor.class);
 
+    private HttpExecutor() {
```
Comment (ned29) [inline src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java]: if you haven't others constructors, constructor by default will created automatically, so i think constructor is unneeded here

Comment (Christoffer-Cortes) [reply src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java]: I fixed this in another commit. Just set the class to final and remove the constructor. =)

Comment (tobiasake) [reply src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java]: It is not my code. Comes from rebase and master branch.

Comment (tobiasake) [reply src/functionaltests/java/com/ericsson/ei/utils/HttpExecutor.java]: OK. Will rebase this PR before merge.

**Pull request merged**